docs/user-guide/configuring.md

```diff
@@ -462,7 +462,7 @@ The complete configuration hierarchy, from highest precedence to lowest preceden
 
 If you want to extend a specific configuration file, you can use the `extends` property and specify the path to the file. The path can be either relative or absolute.
 
-The extended configuration provides base rules, which can be overriden by the configuration that references it. For example:
+The extended configuration provides base rules, which can be overridden by the configuration that references it. For example:
```

Comment:
We should probably have a section here that lists the various options for extending (package, file, file types).